troposphere/static/css/app/instance.scss

```diff
@@ -25,6 +25,10 @@
   &.inactive {
     background: crimson;
   }
+
+  &.error {
+    background: crimson;
+  }
```

Comment:
_(feel free to ignore this - just tossing it in with a mention of @Tharon-C to see what he thinks)_

... this is sort of a tangent - but do we think that _inactive_ might be worth differentiating? The default status is `gray`, I know. Maybe a darker grey? 

More likely something for UI / UX discussion at a later point